fastly/block_fastly_service_logging_s3.go

```diff
@@ -446,6 +455,7 @@ func (h *S3LoggingServiceAttributeHandler) buildCreate(s3Map any, serviceID stri
 		BucketName:                   gofastly.String(resource["bucket_name"].(string)),
 		CompressionCodec:             gofastly.String(resource["compression_codec"].(string)),
 		Domain:                       gofastly.String(resource["domain"].(string)),
+		FileMaxBytes:                 gofastly.Int(resource["file_max_bytes"].(int)),
```

Comment:
The zero value for int (`0`) will be wrapped in a pointer and subsequently, zero will be sent to the API. 

Is that OK? e.g. the API will know to ignore zero? (because logically zero doesn't make sense)

Or would it be better to check for `> 0` as we do further down on line 486?

e.g.

```go
if v, ok := resource["file_max_bytes"].(int); v > 0 {
	opts.FileMaxBytes = gofastly.Int(v)
}
```

This way if the value is zero (either because that's what the user set explicitly, or because the value wasn't provided so `0` is the zero value for the `int` type), then we'll end up sending `nil` to go-fastly and it will omit the value from the request body to the API.

Reply:
Yeah, that's a great idea. I'll make that change.